docs/getting_started/faq.rst

```diff
@@ -0,0 +1,110 @@
+
+What’s the issue with pandas? Why should I use Modin?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""
+In pandas, you are only able to use one core at a time when you are doing computation of
+any kind. This approach does not scale on larger data sets and adding more hardware does not
+solve the problem.  
+
+The ``modin.pandas`` `DataFrame`_ is an extremely light-weight parallel DataFrame. Modin
+transparently distributes the data and computation so that all you need to do is
+continue using the pandas API as you were before installing Modin. With Modin, 
+you are able to use all of the CPU cores on your machine, and because of it's light-weight
+nature often results in less memory overhead than pandas.
+
+Why not just improve pandas?
+""""""""""""""""""""""""""""
+pandas is a massive community and well established codebase. Many of the issues
+we have identified and resolved with pandas are fundamental to its current
+implementation. While we would be happy to donate parts of Modin that
+make sense in pandas, many of these components would require significant (or
+total) redesign of the pandas architecture. Modin's architecture goes beyond
+pandas, which is why the pandas API is just a thin layer at the user level. To learn
+more about Modin's architecture, see the architecture_ documentation.
+
+How much faster can I go with Modin compared to pandas?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+Modin is designed to scale with the amount of hardware available.
+Even in a traditionally serial task like ``read_csv``, we see large gains by efficiently 
+distributing the work across your entire machine. Because it is so light-weight, 
+Modin provides speed-ups of up to 4x on a laptop with 4 physical cores. This speedup scales
+efficiently with larger machines with more cores. We have several published papers that
+include performance results and comparisons against pandas.
+
+How much more data would I be able to process with Modin?
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+We have focused heavily on bridging the solutions between DataFrames for small 
+data (e.g. pandas) and large data. Often data scientists require different tools 
+for doing the same thing on different sizes of data. The DataFrame solutions that 
+exist for 1MB do not scale to 1TB+, and the overheads of the solutions for 1TB+ 
+are too costly for datasets in the 1KB range. With Modin, because of its light-weight, 
+robust, and scalable nature, you get a fast DataFrame at 1MB and 1TB+.
+
+How does Modin work under the hood?
+""""""""""""""""""""""""""""""""""""
+Modin is logically separated into different layers that represent the hierarchy of a 
+typical Database Management System. User queries which perform data transformation, 
+data ingress or data egress pass through the Modin query compiler which translates 
+queries from the Pandas API Layer and sends them to the Modin Core DataFrame. The Modin
```

Comment:
The first two sentences here don't make a lot of sense to a lay reader.

Reply:
```suggestion
queries from the pandas API Layer and sends them to the Modin Core DataFrame. The Modin
```